host-controller/src/test/java/org/jboss/as/domain/controller/operations/ReloadRequiredServerTestCase.java

```diff
@@ -518,6 +522,10 @@ public void execute(ModelNode operation, OperationMessageHandler handler, ProxyO
         private OperationStepHandler nextStep;
         protected MockOperationContext(final Resource root, final boolean booting, final PathAddress operationAddress) {
             super(root, booting, operationAddress);
+            Set<RuntimeCapability> capabilities = new HashSet<>();
```

Comment:
That goes back to my concern in the `DefaultCapabilityReferenceRecorder` implementation that now requires to get info from the MRR.

I am not sure if that's a constraint that will always be there. (it used to be absent in this test fixture for example).

Reply:
Supplying capabilities to the add/remove operation handler is redundant.